app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -69,22 +69,26 @@ data class RecentPatient(
             SELECT MAX(updatedAt) latestUpdatedAt, T.*
```

Comment:
I see that there are no tests added for excluding deleted `PrescribedDrug`, `Appointment`, `Communication` and `MedicalHistory`.
Is there a reason for that?